code/modules/research/relics/effects_passive.dm

```diff
@@ -0,0 +1,245 @@
+	firstname = list("statis","cryo","nanofrost","hyperfreeze","modulated","temporal","stabilized")
+
+/datum/relic_effect/reagents/no_react/apply(obj/item/A)
+	..()
+	if(A.reagents)
+		A.reagents.set_reacting(FALSE)
+
+/datum/relic_effect/reagents/big_beaker
+	weight = 20
+	firstname = list("huge","ginormous","deep","singularity","bluespace")
+	var/volume
+
+/datum/relic_effect/reagents/big_beaker/init()
+	volume += rand(1,100)
+	if(prob(60))
+		volume *= rand(2,10)
+	else if(prob(5)) //The holy grail rofl
+		volume = volume ** rand(2,4)
+
+/datum/relic_effect/reagents/big_beaker/apply(obj/item/A)
+	..()
+	if(!A.reagents)
+		return
+	A.reagents.maximum_volume = volume
+
+/datum/relic_effect/reagents/fill_beaker
+	weight = 20
+
+/datum/relic_effect/reagents/fill_beaker/apply(obj/item/A)
+	if(!A.reagents)
+		return
+	var/times = min(rand(1,10),A.reagents.maximum_volume)
+	for(var/count in 1 to times)
+		A.reagents.add_reagent(get_random_reagent_id(),rand(1,round(A.reagents.maximum_volume / times)))
+
+//Supermatter for experimentor reaction.
+/datum/relic_effect/passive/supermatter
+	firstname = list("supermatter")
+	hogged_signals = list(COMSIG_ITEM_EQUIPPED,COMSIG_PARENT_ATTACKBY)
+
+/datum/relic_effect/passive/supermatter/apply_to_component(obj/item/A,datum/component/relic/comp)
+	comp.RegisterSignal(COMSIG_ITEM_EQUIPPED, CALLBACK(src, .proc/touch, A, null))
+	comp.RegisterSignal(COMSIG_PARENT_ATTACKBY, CALLBACK(src, .proc/touch, A))
+
+/datum/relic_effect/passive/supermatter/apply(obj/item/A)
+	A.color = list(LUMA_R,LUMA_R,0,0, LUMA_G,LUMA_G,0,0, LUMA_B,LUMA_B,0,0, 0,0,0,1, 0.5,0.5,0.5,0)
+	A.set_light(l_range = 4)
+	A.resistance_flags |= INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF | FREEZE_PROOF
+	A.AddComponent(/datum/component/radioactive, 400)
+
+/datum/relic_effect/passive/supermatter/proc/touch(obj/item/A,obj/item/W,mob/living/user)
+	if(istype(W, /obj/item/hemostat/supermatter) || istype(W, /obj/item/scalpel/supermatter) || istype(W, /obj/item/nuke_core_container/supermatter))
+		return //Phew
+	radiation_pulse(user, 500, 2)
+	playsound(get_turf(user), 'sound/effects/supermatter.ogg', 50, 1)
+	if(!W)
+		to_chat(user, "<span class='warning'>You reach for \the [A] with your hands. That was dumb.</span>")
+		user.dust()
+	else
+		to_chat(user, "<span class='notice'>As it touches \the [A], \the [W] bursts into dust!</span>")
+		qdel(W)
+
+//Sellable for lodsofemone at cargo
+/datum/relic_effect/passive/loadsadosh
+	weight = 20
+	hint = list("The on-board appraisal routine has run out of memory while processing this object.")
+	firstname = list("expensive","commercially-viable","nanocorp","THE ARM","syndi")
+	var/value
+	var/datum/export/relic/export_datum
+
+/datum/relic_effect/passive/loadsadosh/init()
+	value = rand(1,5000)
+	if(prob(40)) //Sometimes it's worth even more dosh
+		value *= rand(1,25)
+
+/datum/relic_effect/passive/loadsadosh/apply(obj/item/A)
```

Comment:
i suggest testing this passive a bit since iirc i couldn't get it to properly work when i worked on this

Reply:
tested it a few rounds ago, doesnt work, I'll fix that eventually